### ISSUE TITLE
Store geojson of the locations on the ES index

### ISSUE THREAD

**Issue by flaneuse** [SuLab/outbreak.info#135]
See #134 

Currently, the choropleths at outbreak.info/maps are loaded as [geojsons](https://github.com/SuLab/outbreak.info/tree/master/web/src/assets/geo) by the front-end and then merged with the result of an API call client-side. Obviously, this puts a load on the front-end, and the join can't be cached for subsequent calls.

1. Investigate the performance hit on the API by storing this additional info on the backend and the query time on the front-end vs. the current behavior.
2. If it looks good: add the geojsons for each region to every data point.
3. Adjust the front-end to query rather than run the join client-side.

**Comment by gkarthik** [comment]
Would attaching a geoJSON record to every record at all admin levels on the api along with #133 fix this issue?

**Comment by flaneuse** [comment]
Yep, that's the idea!  I just don't know if the retrieval time of each geojson is faster than loading the geojson once on the front-end and then doing a front-end join.  i assume so...
